File: TouchPOS_API/Models/USERPROFILE_Mmodels.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TouchPOS_API.Models
{
    public class USERPROFILE_Mmodels
    {

        public class USERPROFILE_Data
        {
            public string UID { get; set; }
            public string UFULLNAME { get; set; }
            public string UPASSWORD { get; set; }
            public string ULEVEL { get; set; }
            public string USTATUS { get; set; }
            public string UREMARK { get; set; }
            public string UUID { get; set; }
            public DateTime date_now { get; set; }
        }

    }
}

File: Blackoffice/Models/USERPROFILE_Mmodels.vb
Imports MD
Public Class USERPROFILE_Mmodels

    Public Class value
        Public Respon As New ResponseModels
        Public Data As List(Of USERPROFILE_Data)
    End Class

    Public Class USERPROFILE_Data

        Public Property UID As String
        Public Property UFULLNAME As String
        Public Property UPASSWORD As String
        Public Property ULEVEL As String
        Public Property USTATUS As String
        Public Property UREMARK As String
        Public Property UUID As String
        Public Property date_now As DateTime
    End Class


End Class
